src/test/java/uk/gov/hmcts/probate/service/evidencemanagement/upload/EmUploadServiceTest.java

```diff
@@ -70,4 +72,15 @@ public void shouldStoreFile() throws Exception {
         verify(httpHeadersFactory).getMultiPartHttpHeader();
         verify(documentManagementURIBuilder).buildUrl();
     }
+
+    @Test
+    public void testExceptionWithNullFromApiCall() {
+        when(documentManagementURIBuilder.buildUrl()).thenReturn(URL);
+        when(evidenceManagementRestTemplate.postForObject(eq(URL), any(), eq(HashMap.class))).thenReturn(null);
+        EvidenceManagementFileUpload evidenceManagementFileUpload =
+            new EvidenceManagementFileUpload(MediaType.APPLICATION_PDF, new byte[100]);
+
+        Assertions.assertThatThrownBy(() -> emUploadService.store(evidenceManagementFileUpload))
+            .isInstanceOf(ClientDataException.class);
+    }
```

Comment:
why dont you use the same test structure here as you do with the FeeServiceTest below?